config/daemonconfig/fscache.go

```diff
@@ -38,9 +45,9 @@ type FscacheDaemonConfig struct {
 		CacheConfig struct {
 			WorkDir string `json:"work_dir"`
 		} `json:"cache_config"`
-		MetadataPath string `json:"metadata_path"`
+		PrefetchConfig BlobPrefetchConfig `json:"prefetch_config"`
```

Comment:
Should name the field `BlobPrefetchConfig` ?

Reply:
fixed